src/hazelweb/gui/Page.re

```diff
@@ -56,6 +56,22 @@ let cell_status_panel = (~settings: Settings.t, ~model: Model.t, ~inject) => {
           ),
         ],
       ),
+      div(
+        [Attr.classes(["result-control"])],
+        [
+          {
+            Node.button(
+              [
+                Attr.id("serialize-DHExp"),
+                Attr.on_click(_ =>
+                  inject(SerializeToConsole(DHExp(result)))
+                ),
+              ],
+              [Node.text("Serialize result to console")],
```

Comment:
why not put this in the dropdown menu alongside the other serialize option?

Reply:
https://github.com/hazelgrove/hazel/pull/531/commits/84756c308186ed12e9cb08399a62ec82db89c476

@coned I sort of did something similar for Zexp (just for my visualization purposes), but I think this is similar to what @cyrus- means (i.e., adding as ActionMenu item)?

Reply:
One reason I added this menu was that I was hoping it would provide an unobtrusive place for people to add stuff they develop for their own debugging purposes, so if anyone has something they think might be more generally useful, please do suggest it or make a PR!
![2021-08-09 13_08_56-Hazel](https://user-images.githubusercontent.com/22436459/128746187-033357dc-887b-4ba2-8ec8-95cbc0e37288.png)

Reply:
Yup, let's put debugging things like this up there.